contrib/kind-common

```diff
@@ -751,6 +751,13 @@ deploy_bgp_external_server() {
     echo "FRR kind network IPv6: ${bgp_network_frr_v6}"
     $OCI_BIN exec bgpserver ip -6 route replace default via "$bgp_network_frr_v6"
   fi
+  # disable the default route to make sure the container only routes accross
+  # directly connected or learnt networks (doing this at the very end since
+  # docker changes the routing table when a new network is connected)
+  docker exec frr ip route delete default
```

Comment:
don't we want to gate this with if  [ "$PLATFORM_IPV6_SUPPORT" == true ] ; then and if  [ "$PLATFORM_IPV4_SUPPORT" == true ] ; then? based on kind.sh -i6 -n4 flags?

Reply:
No need. Both stacks are enabled anyway and the command doesn't fail if the route doesn't exist.